client/src/components/MenuIngame.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Classes, Icon, Menu, MenuDivider, MenuItem } from '@blueprintjs/core';
```

Comment:
A ce le noté mais il faudra wrapper chaque composant si ils sont réutilisé quelque part

Reply:
@AuxB Ca je comprends pas